packages/elements-react/src/locales/no.json

```
@@ -0,0 +1,285 @@
+  "logout.reject-button": "Nei",
+  "logout.title": "Ønsker du å logge ut?",
+  "recovery.login-button": "Logg inn",
+  "recovery.login-label": "Husker du legitimasjonen din?",
+  "recovery.title": "Gjenopprett kontoen din",
+  "recovery.subtitle": "Skriv inn e-postadressen tilknyttet kontoen din for å motta en engangskode",
+  "registration.login-button": "Logg inn",
+  "registration.login-label": "Har du allerede en konto?",
+  "registration.subtitle-oauth2": "For å autentisere {clientName}",
+  "registration.title": "Registrer en konto",
+  "registration.subtitle": "Registrer deg med {parts}",
+  "settings.navigation-back-button": "Tilbake",
+  "settings.navigation-backup-codes": "2FA backupkoder",
+  "settings.navigation-logout": "Logg ut",
+  "settings.navigation-oidc": "Sosial innlogging",
+  "settings.navigation-password": "Passord",
+  "settings.navigation-profile": "Profil",
+  "settings.navigation-totp": "Autentiseringsapp",
+  "settings.navigation-webauthn": "Maskinvaretokener",
+  "settings.navigation-passkey": "Passnøkler",
+  "settings.subtitle-instructions": "Her kan du administrere innstillinger relatert til kontoen din. Vær oppmerksom på at enkelte handlinger krever at du autentiserer deg på nytt.",
+  "settings.subtitle": "Oppdater kontoinnstillingene dine",
+  "settings.title-lookup-secret": "Administrer 2FA backup gjenopprettingskoder",
+  "settings.title-navigation": "Kontoinnstillinger",
+  "settings.title-oidc": "Sosial innlogging",
+  "settings.title-password": "Endre passord",
+  "settings.title-profile": "Profilinnstillinger",
+  "settings.title-totp": "Administrer 2FA TOTP-autentiseringsapp",
+  "settings.title-webauthn": "Administrer maskinvaretokener",
+  "settings.title-passkey": "Administrer passnøkler",
+  "settings.title": "Kontoinnstillinger",
+  "settings.navigation.title": "Kontoinnstillinger",
+  "settings.password.title": "Endre passord",
+  "settings.password.description": "Endre passordet ditt",
+  "settings.profile.title": "Profilinnstillinger",
+  "settings.profile.description": "Oppdater profilinformasjonen din",
+  "settings.webauthn.title": "Administrer maskinvaretokener",
+  "settings.webauthn.description": "Administrer innstillingene for maskinvaretokener",
+  "verification.registration-button": "Registrer deg",
+  "verification.registration-label": "Har du ikke en konto?",
+  "verification.title": "Bekreft kontoen din",
+  "verification.subtitle": "Skriv inn e-postadressen tilknyttet kontoen din for å bekrefte den",
+  "verification.back-button": "Tilbake",
+  "two-step.password.title": "Passord",
+  "two-step.password.description": "Skriv inn passordet tilknyttet kontoen din",
+  "two-step.code.title": "E-postkode",
+  "two-step.code.description": "En verifiseringskode vil bli sendt til din e-post",
+  "two-step.webauthn.title": "Sikkerhetsnøkkel",
+  "two-step.webauthn.description": "Bruk sikkerhetsnøkkelen din for å autentisere",
+  "two-step.passkey.title": "Passnøkkel (anbefalt)",
```

```suggestion
  "two-step.passkey.title": "Passkey (anbefalt)",
```

Passkey is a recognized term, and shouldn't be translated, IMO. WDYT?

---

I'm not sure here, I just looked up what Google was doing and they were translating it to "Passnøkkel": https://support.google.com/accounts/answer/13548313?hl=no

I also see that Microsoft isn't very consistent with it, they use "tilgangsnøkkel", "adgangsnøkkel" and "passkey" in norwegian.

Keeping it as "Passkey" seems like the best option.

---

updated the remaining here: https://github.com/ory/elements/pull/544/commits/a7266089ef9984485a0441b1731f262f3bb6857f

also ran make format to satisfy the CI: https://github.com/ory/elements/pull/544/commits/5ace01d01b296e72f08ffbd3986c4c5522c3913e

anything more you need from me on this PR?